app/models/institutional-user.ts

```diff
@@ -0,0 +1,21 @@
+import { attr, belongsTo } from '@ember-decorators/data';
+import DS from 'ember-data';
+
+import UserModel from 'ember-osf-web/models/user';
+import OsfModel from './osf-model';
+
+export default class InstitutionalUserModel extends OsfModel {
+    @attr('string') department?: string;
```

Comment:
```suggestion
    @attr('fixstring') department?: string;
```
Our backend is not great when it comes to cleaning up malformed text.